docs/mining/proof-of-stake.md

```diff
@@ -57,7 +57,7 @@ These stake pools are officially recognized:
 * [<i class="fa fa-external-link-square"></i> https://stakepool.dcrstats.com](https://stakepool.dcrstats.com)
 * [<i class="fa fa-external-link-square"></i> https://stake.decredbrasil.com](https://stake.decredbrasil.com)
 * [<i class="fa fa-external-link-square"></i> http://stakepool.eu](http://stakepool.eu)
-* [<i class="fa fa-external-link-square"></i> http://dcrstakepool.getjumbucks.com](http://dcrstakepool.getjumbucks.com)
+* [<i class="fa fa-external-link-square"></i> https://dcr.ubiqsmart.com - formerly Jumbucks Stake Pool](https://dcr.ubiqsmart.com)
```

Comment:
Can you change the http:// links to https://? All the pools support it now.  Also I don't think the 'formerly' is needed since presumably the admin would notify existing users of the URL change.

And finally, please add a fixes/closes #71 note since there's an issue open on this.